qa/tasks/cephfs/test_fstop.py

```diff
@@ -81,6 +81,9 @@ def test_dumpfs(self):
         """
         Tests 'cephfs-top --dumpfs' output is valid
         """
+        # disable stats plugin for unnecessary notifys
+        self._disable_mgr_stats_plugin()
```

Comment:
I don't have anything against this optimization, but is this _really_ required?

Reply:
Fixed